demo/events.js

```diff
@@ -0,0 +1,80 @@
+var map = L.map('map').setView([39.74739, -105], 13);
```

Comment:
I think a cypress test for all events would be great too (at least all events that can be tested)